### DISCUSSION TITLE
Docker Hub version vs latest release

### DISCUSSION THREAD

**Discussion by acseven** [Budibase/budibase#9207] [Help] [answered]
Hi, 

Docker hub image (latest, 14th Dec) seems to be behind the latest release (19th Dec). Is there any specific reason for this?

![image](https://user-images.githubusercontent.com/1176371/210421057-75849d36-d7ad-4e60-a0ad-02469affaaa1.png)

![image](https://user-images.githubusercontent.com/1176371/210421193-11574c74-5395-4b9d-b3e7-5501185331f1.png)

**Comment by melohagan** [comment, marked as answer]
Hey @acseven 

The single image build had failed, but I re-ran it and it appears to have successfully deployed now: https://hub.docker.com/r/budibase/budibase

![Screenshot 2023-01-04 at 13 57 50](https://user-images.githubusercontent.com/101575380/210570804-37259296-bf31-49f8-8d76-a6d15373b078.png)

**Reply by acseven** [reply]
Great, thanks!
